doc/examples/config/config-storage/README.md

```diff
@@ -71,6 +71,12 @@ The data corresponding to the path contains the following fields:
 * `value`: any string.
 * `mod_revision`: unsigned number, last `revision` at which this
 value modification occurred.
+* An optional time until expiration. After this time mark, the data is
+  considered as deleted (since Tarantool 3.2).
+
+Since Tarantool version 3.2 keys could be added to the storage with the
+specified TTL. The keys guaranteed not to expire early, but they could live a
+bit longer than expected due to communications between replicas and master.
```

Comment:
(Sorry that I can't send all the comments at once: some thoughts arrive into the head after some delay.)

I would highlight that the expiration bumps the storage revision and wake ups watchers.

It is similar to what etcd docs [states](https://etcd.io/docs/v3.4/learning/api/#lease-api) regarding the lease mechanism:

> Each expired key generates a delete event in the event history.

Reply:
Updated.